sign/schnorr/schnorr_test.go

```diff
@@ -62,3 +65,34 @@ func TestEdDSACompatibility(t *testing.T) {
 	}
 
 }
+
+// Simple random stream using the random instance provided by the testing tool
+type quickstream struct {
+	rand *rand.Rand
+}
+
+func (s *quickstream) XORKeyStream(dst, src []byte) {
+	s.rand.Read(dst)
+}
+
+func (s *quickstream) Generate(rand *rand.Rand, size int) reflect.Value {
+	return reflect.ValueOf(&quickstream{rand: rand})
+}
+
+func TestQuickSchnorrSignature(t *testing.T) {
+	f := func(rand *quickstream, msg []byte) bool {
+		suite := edwards25519.NewBlakeSHA256Ed25519WithRand(rand)
```

Comment:
why do you need `rand`? can't you just use `edwards25519.NewBlakeSHA256Ed25519`?

Reply:
It was in order to use the random instance provided by quick. As the key length is fixed, I could remove it and generate only the message.